doc/reference/configuration/configuration_reference.rst

```diff
@@ -3599,6 +3600,290 @@ The ``replication`` section defines configuration parameters related to :ref:`re
     | Default: ``false``
     | Environment variable: TT_REPLICATION_ANON
 
+.. _configuration_reference_replication_autoexpel:
```

Comment:
Changes from another PR. Please separate the changes related to the current task only.